src/System.Collections.Concurrent/tests/ConcurrentDictionary/ConcurrentDictionaryTests.cs

```diff
@@ -546,25 +546,51 @@ public static void TestDebuggerAttributes()
         }
 
         [Fact]
-        public static void TestConstructor_Negative()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, ".NET Framework hasn't received the fix for https://github.com/dotnet/corefx/issues/18432 yet.")]
+        public static void TestNullComparer()
```

Comment:
But it's not necessarily using the right comparer to get it. I don't think there's a way via public surface area to validate 100% that is the right comparer, but a few items could be added, the result of TryAdd validated to match expectations, etc. For strings you could verify that it's doing a case sensitive comparison. And so on. Doesn't need to be much.